__tests__/app.test.js

```diff
@@ -170,6 +169,151 @@ describe("get comments by article id", () => {
+            body: "Made pasta today",
+            votes: expect.any(Number),
+            author: "butter_bridge",
+            article_id: 2,
+            created_at: expect.any(String),
+          })
+        );
+      });
+  });
+
+  it("400: Content in request body has missing property", () => {
+    const newComment = {
+      username: "butter_bridge",
+    };
+    return request(app)
+      .post("/api/articles/3/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Request isn't formatted correctly");
+      });
+  });
+
+  it("400: Non string is given as username ", () => {
+    const newComment = {
+      username: 1,
+      body: "Pizza",
+    };
+    return request(app)
+      .post("/api/articles/3/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Request isn't formatted correctly");
+      });
+  });
+
+  it("400: Bad path", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "lala",
+    };
+    return request(app)
+      .post("/api/articles/apple/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad request");
+      });
+  });
+
+  it("404: invalid article id", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "hi",
+    };
+    return request(app)
+      .post("/api/articles/3048432/comments")
+      .send(newComment)
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Article not found");
+      });
+  });
+});
+
+describe("Patch /api/articles/:article_id", () => {
+  it("200: responds with article with updated vote", () => {
+    const updatedVote = { inc_votes: 500 };
+
+    return request(app)
+      .post("/api/articles/1")
+      .send(updatedVote)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article[0].votes).toBe(600);
+      });
+  });
+
+  it("404: responds with message if article id doesn't exist", () => {
+    const updatedVote = { inc_votes: 500 };
+    return request(app)
+      .post("/api/articles/3048")
+      .send(updatedVote)
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Article not found");
+      });
+  });
+
+  it("400: responds with message if request body doesn't have integer", () => {
+    const updatedVote = { inc_votes: "banana" };
+    return request(app)
+      .post("/api/articles/1")
+      .send(updatedVote)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad request");
+      });
+  });
+
+  it("400: if body is empty, returns article", () => {
+    const updatedVote = {};
+    return request(app)
+      .post("/api/articles/1")
+      .send(updatedVote)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Request not formatted correctly");
```

Comment:
This message could easily be the same as the above 400 test because both are not formatted correctly and are Bad Requests. I reckon make the messages consistent